build.sbt

```diff
@@ -582,6 +582,7 @@ lazy val tests =
     )
     .settings(noPublishSettings)
     .settings(
+      nativeConfig ~= (_.withMode(scalanative.build.Mode.releaseFast)),
```

Comment:
Hum, was this here before and I missed it? It doesn't seem like this should be part of this PR.

Reply:
Good you've spotted it! I've probably added them by mistake after switching from other branch